README.md

```diff
@@ -1,623 +1,1519 @@
-
 ```
-<!-- - **Windows**
 
-```
+ - **Windows**
+
+  
 
+```
 mkdir sample-data\user 2>nul & ^
 
-curl -L "https://raw.githubusercontent.com/ELEVATE-Project/samiksha-service/main/documentation/1.0.0/sample-data/windows/user/sampleData.sql" -o sample-data\user\sampleData.sql
+curl -L https://raw.githubusercontent.com/ELEVATE-Project/samiksha-service/main/documentation/1.0.0/sample-data/windows/user/sampleData.sql -o sample-data/user/sampleData.sql
+```
 
---> ```
 2.  **Download The `insert_sample_data` Script File:**
 
   
 
+  
+
 -  **Ubuntu/Linux/Mac**
 
-```
+  
 
+```
 curl -L -o insert_sample_data.sh https://raw.githubusercontent.com/ELEVATE-Project/samiksha-service/main/documentation/1.0.0/dockerized/scripts/mac-linux/insert_sample_data.sh && chmod +x insert_sample_data.sh
-
 ```
-<!-- - **Windows**
 
-```
+- **Windows**
 
-curl -L -o insert_sample_data.bat https://raw.githubusercontent.com/ELEVATE-Project/samiksha-service/main/documentation/2.6.1/dockerized/scripts/windows/insert_sample_data.bat
+  
 
 ```
+curl -L -o insert_sample_data.bat https://github.com/ELEVATE-Project/samiksha-service/raw/main/documentation/1.0.0/dockerized/scripts/windows/insert_sample_data.bat
+```
+
+  
 
--->
+
+
+  
 
   
 
 3.  **Run The `insert_sample_data` Script File:**
 
   
 
+  
+
 -  **Ubuntu/Linux/Mac**
 
-```
+  
 
+```
 ./insert_sample_data.sh user postgres://postgres:postgres@citus_master:5432/user
-
 ```
-<!-- - **Windows**
 
-```
+ - **Windows**
 
-insert_sample_data.bat user postgres://postgres:postgres@citus_master:5432/user
+  
 
 ```
+insert_sample_data.bat user postgres://postgres:postgres@citus_master:5432/user
+```
 
--->
+  
+
+
+  
 
   
 
 After successfully running the script mentioned above, the following user accounts will be created and available for login:
 
   
 
+  
+
 | Email ID | Password | Role |
 
+  
+
 | ------------------------ | ---------- | ----------------------- |
 
+  
+
 | aaravpatel@example.com | Password1@ | State Education Officer |
 
+  
+
 | arunimareddy@example.com | Password1@ | State Education Officer |
 
+  
+
 | aaravpatel@example.com | Password1@ | State Education Officer |
 
+  
+
 ## Insert Initial Data
 
--  **Ubuntu/Linux**
+  
 
-  ```
-docker exec -it samiksha sh -c "node documentation/1.0.0/dockerized/scripts/mac-linux/insert_sample_solutions.js"
-  ```
+-  **Ubuntu/Linux** && **Windows**
 
+  
 
-## Sample Data Creation For Projects
+```
+docker exec -it samiksha sh -c "node documentation/1.0.0/dockerized/scripts/mac-linux/insert_sample_solutions.js"
+```
 
->  **Warning:** upload related apis will not work because cloud integration is not enabled in this set-up.
+  
+ 
 
   
 
 </details>
 
   
 
+  
+
 <details>
 
+  
+  
 
   
 
 </details>
 
-```sql
+  
+
+
+<summary>Natively Installed Services & Dependencies </summary>
```

Comment:
@borkarsaish65  it is showing as wrong in preview fix it

Reply:
Fixed it